packages/@uppy/provider-views/src/ProviderView/User.tsx

```diff
@@ -3,15 +3,17 @@ import { h, Fragment } from 'preact'
 type UserProps = {
   i18n: (phrase: string) => string
   logout: () => void
-  username: string | undefined
+  username: string | null
 }
 
 export default function User({ i18n, logout, username }: UserProps) {
   return (
     <Fragment>
-      <span className="uppy-ProviderBrowser-user" key="username">
-        {username}
-      </span>
+      {username && (
```

Comment:
**EXPLANATION**

This `username &&` thing fixes the issue in some providers that would be showing a small dot even when we don't have the username (doesn't look good).